thunder/cli/backend.py

```diff
@@ -114,6 +108,23 @@ def populate(backend_name):
     return backend_params
 
 
+def collect_backends() -> Dict[str, Backend]:
+    configs, _ = collect_configs()
+    local_backends = {name: backends[config.backend] for name, config in configs.items()}
+    return {**backends, **local_backends}
```

Comment:
use a ChainMap or update the copy. I'm not sure in which python version this expansion started to be unambiguous